hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/map/MapAssignAndGetUuidsOperation.java

```diff
@@ -23,16 +23,16 @@
 import com.hazelcast.map.impl.MapServiceContext;
 import com.hazelcast.map.impl.nearcache.MapNearCacheManager;
 import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+import com.hazelcast.spi.impl.operationservice.MutatingOperation;
 import com.hazelcast.spi.impl.operationservice.Operation;
 import com.hazelcast.spi.impl.operationservice.PartitionAwareOperation;
-import com.hazelcast.spi.impl.AllowedDuringPassiveState;
 
 import java.util.UUID;
 
 import static com.hazelcast.map.impl.MapService.SERVICE_NAME;
 
 public class MapAssignAndGetUuidsOperation extends Operation
-        implements PartitionAwareOperation, IdentifiedDataSerializable, AllowedDuringPassiveState {
```

Comment:
Out of curiosity, why isn't `AllowedDuringPassiveState` necessary any more (or was it ever necessary in the first place)?

Reply:
Maybe it should have been addressed in a separate PR. But, during cleanup, i encountered that operation which was missing from the test scope. Found the issue and embedded the fix here. 

In short, no need `AllowedDuringPassiveState` for that mutating operation.